processing file asmFiles/test.branchjump1.asm (1 of 6)
processing file asmFiles/test.ldst1.asm (2 of 6)
processing file asmFiles/test.ldst2.asm (3 of 6)
processing file asmFiles/test.loadstore.asm (4 of 6)
processing file asmFiles/test.rtype1.asm (5 of 6)
processing file asmFiles/test.rtype.asm (6 of 6)
processing file asmFiles/mult.asm (1 of 1)
processing file asmFiles/fib.asm (1 of 1)
processing file asmFiles/search.asm (1 of 1)
--- memsim.hex	2014-10-08 20:47:39.399103747 -0400
+++ memcpu.hex	2014-10-08 20:47:39.379103194 -0400
@@ -14,7 +14,7 @@
 :04000D001000FFF9E7
 :04000E00AFA500009A
 :04000F00FFFFFFFFF1
-:0400200000000124B7
+:040020000000008C50
 :0400210000005C6F10
 :040022000000006476
 :040023000000087D54
